src/app/course-item/course-item.component.html

```diff
@@ -25,14 +25,11 @@ <h2 class="header__title">
     <div class="row content">
       <p class="content__description">{{ course.description }}</p>
       <div class="content__actions">
-        <button class="action-button">
+        <button class="action-button" (click)="editItem(course)">
```

Comment:
you can use [RouterLink](https://angular.io/api/router/RouterLink) but this approach is ok too.